example/analysis_options.yaml

```diff
@@ -22,8 +22,11 @@ linter:
   # `// ignore_for_file: name_of_lint` syntax on the line or in the file
   # producing the lint.
   rules:
-    # avoid_print: false  # Uncomment to disable the `avoid_print` rule
-    # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
+  # avoid_print: false  # Uncomment to disable the `avoid_print` rule
+  # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
 
 # Additional information about this file can be found at
 # https://dart.dev/guides/language/analysis-options
+analyzer:
+  errors:
+    deprecated_member_use: ignore
```

Comment:
Should not be necessary for Example, right? We are using `xxxPreview` currently

Reply:
The Drop-in example still uses the deprecated object. There will be another PR for the adjustments regarding drop-in. Then we should be able to remove the rule again.